index.js

```diff
@@ -33,6 +33,11 @@ export default {
       type: Object,
       default: () => ({}),
     },
+    templateComponents: {
+      type: Object,
+      default: () => ({}),
+    },
+    scoped: Boolean,
```

Comment:
Shall we add a default to this? I'm thinking a default of `false`, then it's backward compatible.